daru-io.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'dbi'
   spec.add_development_dependency 'jsonpath'
   spec.add_development_dependency 'mechanize'
+  spec.add_development_dependency 'mongo'
```

Comment:
(Changes for Optional dependency is planned to be made later, according to discussion on issue #25 )